exporter/signalfxexporter/README.md

```diff
@@ -114,6 +114,80 @@ One of `realm` and `api_url` are required.
   - `cleanup_interval` (default = 1 minute): How frequently to purge duplicate requests.
   - `sync_attributes` (default = `{"k8s.pod.uid": "k8s.pod.uid", "container.id": "container.id"}`) Map containing key of the attribute to read from spans to sync to dimensions specified as the value.
 
+## Default Metric Filters
+
+By default, several metrics excluded when sending data through the SignalFx exporter. If `translation_rules` are configured and new metrics match a default exclusion, the new metric will still be excluded. Some metrics are only excluded if specific resource labels (dimensions) are present. Some exclusion rules use regex to exclude multiple metric names.
```

Comment:
Do you mind explaining why this is the case?

Reply:
I decided to re-write this whole section after you mentioned this @rmfitzpatrick , thanks.

Reply:
@rmfitzpatrick I can merge if you are ok with the change.